src/couch/src/couch_db.erl

```diff
@@ -414,37 +390,34 @@ get_before_doc_update_fun(#db{before_doc_update = Fun}) ->
 get_committed_update_seq(#db{committed_update_seq=Seq}) ->
     Seq.
 
-get_update_seq(#db{update_seq=Seq})->
-    Seq.
+get_update_seq(#db{} = Db)->
```

Comment:
Minor nitpick. Some of the `get_` functions return `{ok, Result} and some just `Result`.